pkg/apis/migration/v1alpha1/model.go

```diff
@@ -161,3 +167,114 @@ func GetSecret(client k8sclient.Client, ref *kapi.ObjectReference) (*kapi.Secret
 
 	return &object, err
 }
+
+// DeleteLabels - removes migration  labels from the resource after restore
+func DeleteLabels(config *rest.Config, uid string) error {
```

Comment:
DeleteLabels and DeleteMIgrated follow the same structure. They're basically identical except for the extra foreground/deleteOptions block in DeleteMigrated and the actual label/delete steps within the `range list.Items` loop. It's not a huge deal, but it might be nice if these could be refactored to remove some of the duplication.